### PR TITLE
Fix build and update readme

### PR DESCRIPTION
**Pull request by C0rn3j**
Couldn't get it to package without this on Arch Linux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,32 @@ Exhausts the keyspace on my Ryzen 3700X in 4 minutes and 11 seconds which equals
 
 ## Installation
 
-On Linux: grab a binary from the [releases-page](https://github.com/MaxMatti/aax-bruteforce/releases) or build it yourself, then run it.
+### Linux
+
+Grab a binary from the [releases-page](https://github.com/MaxMatti/aax-bruteforce/releases) or build it yourself, then run it.
+
+People running Arch Linux(or derivatives) can use the [AUR package](https://aur.archlinux.org/packages/aax-bruteforce).
```
Comment (MaxMatti) [inline README.md:13]: That's a 404 for me, do you intend to create this?

File: README.md [hunk 2]
```diff
@@ -6,24 +6,32 @@ Exhausts the keyspace on my Ryzen 3700X in 4 minutes and 11 seconds which equals
...
 
 ## Building
 
-Dependencies: Clang (I used version 11.1.0)
+Dependencies: 
+* Clang - Tested with version 17.0.6
```
Comment (MaxMatti) [inline README.md]:
```suggestion
* Clang - Tested with versions 11.1.0 and 17.0.6
```

File: README.md [hunk 3]
```diff
@@ -6,24 +6,32 @@ Exhausts the keyspace on my Ryzen 3700X in 4 minutes and 11 seconds which equals
...
+
+Building the `main` binary is as simple as cloning the repo and running make:
 ```bash
 git clone --depth=1 https://github.com/MaxMatti/aax-bruteforce.git
+cd aax-bruteforce
```
Comment (MaxMatti) [inline README.md:23]: :heart:

File: main.cpp [hunk 4]
```diff
@@ -4,6 +4,7 @@
 #include <iostream>
 #include <string>
 #include <thread>
+#include <vector>
```
Comment (MaxMatti) [inline main.cpp:7]: I don't know how it worked for me without this :sweat_smile:

**Review by MaxMatti — CHANGES_REQUESTED** [review]
Pls address the AUR question, the other comment is just a nitpick and not necessary for merge.

File: README.md (hunk 1 above)
Comment (C0rn3j) [reply README.md:13]: Yep, working on it right now

**Review by MaxMatti — APPROVED** [review]

**Comment by C0rn3j** [comment]
Thank you, could you tag a release too when you've some spare time so I could fix up the package against it?

**Comment by MaxMatti** [comment]
I'm currently looking into Github Actions (in the background, while doing some other stuff) to automatically create artifacts for a release. If you want to speed the release up you can help me with that, then I can create one.

**Comment by C0rn3j** [comment]
Unfortunately don't have much spare time or experience with GA, I'll be waiting patiently

**Comment by MaxMatti** [comment]
Sorry, I forgot about it. Release is published, I'll look into the automation later when I find the time.

**Pull request merged**